lib/pluginmanager/templates/input-plugin/lib/logstash/inputs/example.rb.erb

```diff
@@ -8,7 +8,7 @@ require "socket" # for Socket.gethostname
 # This plugin is intented only as an example.
 
 class LogStash::Inputs::<%= classify(plugin_name) %> < LogStash::Inputs::Base
-  config_name "<%= @plugin_name %>"
+  config_name "<%= plugin_name %>"
```

Comment:
:+1: could you please maybe also update the other places where the i-var is used instead e.g.
- https://github.com/elastic/logstash/blob/main/lib/pluginmanager/templates/input-plugin/lib/logstash/inputs/example.rb.erb#L11=
- https://github.com/elastic/logstash/blob/main/lib/pluginmanager/templates/codec-plugin/lib/logstash/codecs/example.rb.erb#L10-L16=

Reply:
woops, thought I'd caught them all! I've updated the PR with the usages in the codec example plugin also. Thanks!